otherlibs/stdune/src/import.ml

```diff
@@ -0,0 +1,31 @@
+module Pp = struct
+  include Pp
```

Comment:
Can't this live in its own module?

Reply:
It throws ocamldep into a dependency cycle if I introduce another compilation unit. If I do something like `pp0.ml`, I have to annotate many other places.